BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -98,27 +89,41 @@ extension ElementContent {
         init(layout: LayoutType) {
             self.layout = layout
         }
+        
+        /// Adds the given child element.
+        public mutating func add(element: Element, traits: LayoutType.Traits = LayoutType.defaultTraits, key: AnyHashable? = nil) {
```

Comment:
I think we put `traits` before `element` to match the style for Row and Column, etc where you want to be able to have the nested element last (so that it can span multiple lines without the traits/properties being down at the end). That use-case is less relevant for `ContentStorage`, but it would be nice to keep the consistency I think.